content/docs/en/elements/components/tab-view.md

```diff
@@ -23,17 +23,21 @@ The TabView component allows you to display content grouped into tabs.
 
 ## Props
 
-| name | type | description |
+| Name | Type | Description |
 |------|------|-------------|
-| `selectedIndex` | `Number` | The currently selected tab. Default is `0`
+| `selectedIndex` | `Number` | Gets or sets the currently selected tab. Default is `0`.
+| `tabTextColor` | `Color` | (Style property) Gets or sets the text color of the tabs titles.
+| `tabBackgroundColor` | `Color` | (Style property) Gets or sets the background color of the tabs.
+| `selectedTabTextColor` | `Color` | (Style property) Gets or sets the text color of the selected tab title.
 
 ## Events
 
-| name | description |
+| Name | Description |
 |------|-------------|
-| `selectedIndexChanged`| Emitted when one of the tab-view-item components are tapped
+| `selectedIndexChanged` | Emitted when one of the `<TabViewItem>` components is tapped.
+
+## Native component
 
-## Native Component
 | Android | iOS |
 |---------|-----|
-| android.support.v4.view.ViewPager | UITabBarController
+| [`android.support.v4.view.ViewPager`](https://developer.android.com/reference/android/support/v4/view/ViewPager.html) | [`UITabBarController`](https://developer.apple.com/documentation/uikit/uitabbarcontroller)
```

Comment:
This is GREAT! I wanted these to be links, but never actually got around to add them! 👍